documentation/dhcp_config.md

```diff
@@ -15,9 +15,8 @@ Introduced: v7.0.0
 
 | Name                   | Type          | Default                          | Description                                                         | Allowed Values      |
 | ---------------------- | ------------- | -------------------------------- | ------------------------------------------------------------------- | ------------------- |
-| `comment`              | String        | `nil`                            | Comment to add to the configuration file                            |                     |
 | `ip_version`           | Symbol        | `:ipv4`                          | Select DHCP or DHCPv6 server to configure                           | `:ipv4`, `:ipv6`    |
-| `conf_dir`             | String        | `/etc/dhcp/dhcpd(6).d/classes.d` | Directory to create configuration file in                           |                     |
+| `config_file`          | String        | `/etc/dhcp/dhcpd(6).conf`        | 'The full path to the DHCP server configuration on disk             |                     |
```

Comment:
is there a reason to change this, as this change makes this a major release imo (If someone was using this property and it changed everything would break for them :/ )

Reply:
It's a mistake in the documentation from v7.0.0 being corrected, not a resource property change. If you look at the resource it never had a `conf_dir` property so no-one should be using it 😉.